community/erlang/APKBUILD

```diff
@@ -138,47 +131,47 @@ ssh() { _mv_erlang_lib ssh; depends="$depends erlang-public-key erlang-inets"; }
 ssl() { _mv_erlang_lib ssl; depends="$depends erlang-public-key erlang-inets"; }
 stdlib() { _mv_erlang_lib stdlib; }
 syntax_tools() { _mv_erlang_lib syntax_tools; }
+tftp() { _mv_erlang_lib tftp; }
 tools() { _mv_erlang_lib tools; }
-xmerl() { _mv_erlang_lib xmerl; }
 wx() { _mv_erlang_lib wx; }
+xmerl() { _mv_erlang_lib xmerl; }
 
 dev() {
-	set -x
-	local i= j=
-	depends="$pkgname=$pkgver-r$pkgrel $depends_dev"
-	pkgdesc="$pkgdesc (development files)"
+    set -x
+    local i= j=
+    depends="$pkgname=$pkgver-r$pkgrel $depends_dev"
+    pkgdesc="$pkgdesc (development files)"
 
-	cd "$pkgdir" || return 0
-	local libdirs=usr/
-	[ -d lib/ ] && libdirs="lib/ $libdirs"
-	for i in usr/include usr/lib/pkgconfig usr/share/aclocal\
-			usr/share/gettext usr/bin/*-config	\
-			usr/share/vala/vapi usr/share/gir-[0-9]*\
-			usr/share/qt*/mkspecs			\
-			usr/lib/qt*/mkspecs			\
-			usr/lib/cmake				\
-			$(find . -name include -type d) 	\
-			$(find $libdirs -name '*.[acho]' \
-				-o -name '*.prl' 2>/dev/null); do
-		if [ -e "$pkgdir/$i" ] || [ -L "$pkgdir/$i" ]; then
-			d="$subpkgdir/${i%/*}"	# dirname $i
-			mkdir -p "$d"
-			mv "$pkgdir/$i" "$d"
-			rmdir "$pkgdir/${i%/*}" 2>/dev/null || true
-		fi
-	done
-	# move *.so links needed when linking the apps to -dev packages
-	for i in lib/*.so usr/lib/*.so; do
-		if [ -L "$i" ]; then
-			mkdir -p "$subpkgdir"/"${i%/*}"
-			mv "$i" "$subpkgdir/$i" || return 1
-		fi
-	done
-	return 0
+    cd "$pkgdir" || return 0
+    local libdirs=usr/
+    [ -d lib/ ] && libdirs="lib/ $libdirs"
+    for i in usr/include usr/lib/pkgconfig usr/share/aclocal\
+                         usr/share/gettext usr/bin/*-config  \
+                         usr/share/vala/vapi usr/share/gir-[0-9]*\
+                         usr/share/qt*/mkspecs           \
+                         usr/lib/qt*/mkspecs         \
+                         usr/lib/cmake               \
+                         $(find . -name include -type d)     \
+                         $(find $libdirs -name '*.[acho]' \
+                                -o -name '*.prl' 2>/dev/null); do
+        if [ -e "$pkgdir/$i" ] || [ -L "$pkgdir/$i" ]; then
+            d="$subpkgdir/${i%/*}"  # dirname $i
+            mkdir -p "$d"
+            mv "$pkgdir/$i" "$d"
+            rmdir "$pkgdir/${i%/*}" 2>/dev/null || true
+        fi
+    done
+    # move *.so links needed when linking the apps to -dev packages
+    for i in lib/*.so usr/lib/*.so; do
+        if [ -L "$i" ]; then
+            mkdir -p "$subpkgdir"/"${i%/*}"
+            mv "$i" "$subpkgdir/$i" || return 1
```

Comment:
If you're improving the APKBUILD anyway, these `|| return 1` can be removed.

Reply:
Point was primarily to get it going for now, but I'll remove it while I'm at it.